### ISSUE TITLE
Fix for AppImages?

### ISSUE THREAD

**Issue by palob** [Foggalong/hardcode-fixer#354]
Applications packaged by means of AppImageKit come with icon line in their ``.desktop`` files which read ``Icon=appimagekit-*iconname*``.

We could either fix it by introducing lots of ``appimagekit-`` symlinks to icon themes or by having hardcode-fixer strip those prefixes. Is there anything unfavourable to this solution?

**Comment by bilelmoussaoui** [comment]
I don't think we should never fix this here. Appimges should stop doing that and use whatever the app developer decided to use an icon name instead of changing things.
